lib/tweet-webhook.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
+const privateConfig = require("../private-config.json");
+const { standardFromRepo } = require("./helpers/workstreams.js");
+const composeTweet = require("./compose-tweet.js");
+const Twitter = require("twitter-lite");
+
+module.exports = async body => {
+  if (body.ref !== "refs/heads/main") {
+    return;
+  }
+
+  const standard = standardFromRepo(body.repository.name);
+  if (!standard) {
+    return;
+  }
+
+  const { twitter } = standard;
+  const keys = privateConfig.twitterAccounts[twitter];
+  if (!keys) {
+    return;
```

Comment:
This should probably throw an exception instead, as it indicates a misconfiguration?

Reply:
I was thinking that as we add new standards we might not immediately have this configured.

Reply:
Sure, but then we should have the webhook fail, instead of silently succeeding with a 204 but doing nothing.

Reply:
I guess that's fair, the webhook failing is not the end of the world. Feel free to adjust this.